spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
ah yes, sorry, missed that. ok, so then, why do a custom helper just for this patch? we made the patch directly in the spec file in all the other plugins?